app/Http/Controllers/Auth/LogoutController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\BaseController;
+use Illuminate\Support\Facades\Auth;
+
+class LogoutController extends BaseController
```

Comment:
Can this be an invokable, single method controller?

Reply:
1dc053de